addons/l10n_ec/data/account_tax_report_data.xml

```diff
@@ -333,6 +333,15 @@
                                     </record>
                                 </field>
                             </record>
+                            <!-- TODO ensure line above for v16 matches line below for v15 -->
```

Comment:
@odoostan Do we still have some doubt here?

Reply:
No, @pepetreshere checked and I think fixed (or maybe said it was all right like that) but in any case I'll remove this comment.